superbit_lensing/medsmaker/superbit/medsmaker_mocks.py

```diff
@@ -722,7 +736,7 @@ def _meds_metadata(self,magzp=0.0):
         meta['magzp_ref'] = magzp
         return meta
 
-    def _calculate_box_size(self,angular_size,size_multiplier = 2.5, min_size = 16, max_size= 64, pixel_scale = 0.206):
+    def _calculate_box_size(self,angular_size,size_multiplier = 2.5, min_size = 16, max_size= 64, pixel_scale = 0.144):
```

Comment:
This type of thing makes me nervous...I know that the `pixel_scale` shouldn't be changing for superBIT, but what happens if our code gets used by someone else, or for gigabit, etc.? Though there are probably many instances where this assumption is made 🤷‍♂️

Reply:
Agreed, the fewer hard-coded numbers the better. What do you think would be the best way to handle this? A `BITMeasurement` attribute?

Reply:
Yes, I think that is a good solution

Reply:
I'm working on this right now.

Reply:
OK, I've implemented a solution to this: I added a `get_pixel_scale()` method to `BITMeasurement` that returns the pixel scale of the coadded image or any other supplied image. 

We could eventually adapt this `get_pixel_scale()` method for anyplace else in the pipeline where a pixel scale is hard-coded in (perhaps in `utils.py`?).

Fortunately, this `_calculate_box_size` method is the only place in this code where a pixel scale was explicitly used.

Reply:
above added to `utils.py`